config.yml

```diff
@@ -1,4 +1,6 @@
 prefix: "^"
+privateChannelPrefix: "🔒"
```

Comment:
Instead of prefix/suffix, maybe set to "anywhere in string" and just have an array of options that work

Reply:
Does 375347d024885fb2a32c06665d67c0760e941358 address this enough? I would have to look more into the array option

Reply:
Close enough - I'll handle it myself right now.